modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/service/AnomalyRequest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.modelservice.service;
+
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+public class AnomalyRequest {
+    private String hash;
```

Comment:
This should be `metricId`, not `hash`. The fact that we generate the ID as a hash is an implementation detail. And we'd want it to be `metricId` so that it doesn't sound like anomaly request ID.

But we're getting ready to remove metrics from model service, so we should avoid building metric IDs into the system, that's what this is supposed to represent.

Reply:
Do you mean the metricId which we store in model service DB?

Reply:
See https://graphite.readthedocs.io/en/latest/tags.html

Can we try this?